DFC.ServiceTaxonomy.GraphSync/GraphSyncers/GraphSyncer.cs

```diff
@@ -83,6 +90,24 @@ public async Task SyncToGraph(ContentItem contentItem)
             await SyncComponentsToGraph(graphSyncPartContent);
         }
 
+        public async Task DeleteFromGraph(ContentItem contentItem)
```

Comment:
We should move this into its own delete specific graph sync class (and probably rename this class to something akin to UpsertGraphSyncer or better 😄). They have quite different dependencies.